geonode/maps/api/serializers.py

```diff
@@ -76,15 +80,24 @@ class Meta:
             "current_style",
             "styles",
             "dataset",
+            "stack_order",
```

Comment:
Why are you setting the following fields? This information is inside the blob, and is only required by the client. Why do we need to be aware of these attributes?

Reply:
Changed